dist/locales/en-AU.json

```diff
@@ -633,8 +633,9 @@
   "impac.widget.editable_title.tooltip_text": "(click to edit name)",
   "impac.widget.editable_title.incorrect_name": "Incorrect name",
 
-  "impac.dashboard.create.create_new_dashboard":      "Create New Dashboard",
+  "impac.dashboard.create.create_new_dashboard":      "Create New {name}",
```

Comment:
Oh, I get it now.. as Dashboard is a noun in the context of Impac!, name is uppercase by default (can be downcased when binding the variable to translate).